src/wrapField.tsx

```diff
@@ -72,10 +74,17 @@ export default function wrapField(
     help,
     required,
     description,
+    deprecated,
     ...props
   }: WrapFieldProps,
   children: React.ReactNode,
 ) {
+  const labelIcons = (
+    <>
+    {description && <FieldHintPopover description={description}/>}
+    {deprecated && <FieldDeprecatedPopover/>}
+    </>
+  );
```

Comment:
In this case, we're creating a new `labelIcons` variable on every render, which is not ideal, because it forces additional renders because of this.

Reply:
The alternative is to write this, directly in the `labelIcon` property of the `<FormGroup />` component or leave it as it was before but try to combine the deprecated popover with the hint popover.